packages/admin-stories/src/docs/development/CreateAComponentWithThemeSupport.stories.mdx

```diff
@@ -0,0 +1,110 @@
+import { Meta } from "@storybook/addon-docs/blocks";
+
+<Meta title="Docs/Development/Create a component with theme support" />
+
+# Create a component with theme support
+
+Material-UI supports overriding of styles and default props of components using the theme.  
+Using the same method it should also be possible to override all CometAdmin components.
```

Comment:
```suggestion
Using the same method it should also be possible to override all @comet/admin components.
```

CometAdmin? @comet/admin? Comet Admin? We should use a consistent name in the docs.